tests/cpp_unit_tests/test_math_solver.cpp

```diff
@@ -51,12 +51,12 @@ TEST_CASE("Test block") {
     }
 }
 
-#define CHECK_CLOSE(x, y, tolerance)                    \
-    do {                                                \
-        if constexpr (sym)                              \
-            CHECK(cabs((x) - (y)) < tolerance);         \
-        else                                            \
-            CHECK((cabs((x) - (y)) < tolerance).all()); \
+#define CHECK_CLOSE(x, y, tolerance)                      \
+    do {                                                  \
+        if constexpr (sym)                                \
+            CHECK(cabs((x) - (y)) < (tolerance));         \
+        else                                              \
+            CHECK((cabs((x) - (y)) < (tolerance)).all()); \
```

Comment:
please also use the `break before braces` here:

```suggestion
        if constexpr (sym) {                              \
            CHECK(cabs((x) - (y)) < (tolerance));         \
        }                                                 \
        else {                                            \
            CHECK((cabs((x) - (y)) < (tolerance)).all()); \
        }
```